src/main/java/com/github/stefvanschie/inventoryframework/util/XMLUtil.java

```diff
@@ -70,7 +72,7 @@ public static void loadFieldAttribute(@NotNull Object instance, @NotNull Element
             field.setAccessible(true);
             field.set(instance, value);
         } catch (NoSuchFieldException | IllegalAccessException e) {
-            e.printStackTrace();
+            throw new RuntimeException(e);
```

Comment:
Please change this exception type to `XMLReflectionException`.

Reply:
This code is executed while loading XML files, so shouldn't `XMLLoadException` be used instead?

Reply:
Yeah, sorry, `XMLLoadException` not `XMLReflectionException`.